src/reducers/fields.js

```diff
@@ -8,37 +8,8 @@ import {
   RESET_INFO_FIELD
 } from 'actions'
 
-import {
-  CHECK_TWO_FACTOR_CODE_SUCCESS,
-  DESACTIVATE_2FA_SUCCESS,
-  AUTH_MODE
-} from 'actions/twoFactor'
-
-const composeReducers = (...fns) => {
-  return (state, action) => {
-    for (let i = 0; i < fns.length; i++) {
-      const reducer = fns[fns.length - i - 1]
-      // eslint-disable-next-line no-param-reassign
-      state = reducer(state, action)
-    }
-    return state
-  }
-}
-
-/** Special reducer for the auth_mode value */
-const authModeValue = (state = AUTH_MODE.BASIC, action) => {
-  switch (action.type) {
-    case CHECK_TWO_FACTOR_CODE_SUCCESS:
-      return AUTH_MODE.TWO_FA_MAIL
-    case DESACTIVATE_2FA_SUCCESS:
-      return AUTH_MODE.BASIC
-    default:
-      return state
-  }
-}
```

Comment:
What was the purpose of these functions?

Reply:
Before the refactor of the 2FA part, we managed all requests and their status with redux. We also determined from this status whether 2FA had been activated. Now, we determine directly in the 2FA component whether it is activated or not. I decided to remove this part because it was no longer useful as we no longer have a state in redux